lib/rubocop/cop/style/documentation_method.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8
+# frozen_string_literal: true
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for missing documentation comment for public method.
```

Comment:
Add some examples here of code that would be flagged by the cop. You can see other cops for references - e.g. https://github.com/Drenmi/rubocop/blob/635403972a3b768d9d709f04eed52090d7738d63/lib/rubocop/cop/style/numeric_predicate.rb